zipline/src/commonMain/kotlin/app/cash/zipline/FlowReference.kt

```diff
@@ -30,9 +31,9 @@ class FlowReference<T> @PublishedApi internal constructor(
   @Contextual private val referenceFlowReference: ZiplineReference<ReferenceFlow>,
   @Contextual private val serializer: ZiplineSerializer<T>,
 ) {
-  private fun getJsonFlow(): Flow<String> {
+  @OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
Hmm what is this for? We ideally should not have any of these in library code since it means we're subject to potential binary incompatibility in the future.

Reply:
IntelliJ is giving me warnings for `ProducerScope` which comes from using `channelFlow`.

![Uploading Screen Shot 2021-09-30 at 15.49.27.png…]()

Reply:
We'll have to get rid of `channelFlow { }` then and build it ourselves with a `flow { }` and a channel.